electrum_nmc/electrum/names.py

```diff
@@ -773,6 +779,32 @@ def get_domain_records_address_tor_single(domain, value):
 
     return [domain, "address", ["tor", value]], None
 
+def get_domain_records_address_ipfs(domain, value):
+    # Convert string to array if only 1 IPFS record exists
```

Comment:
Is there ever a valid reason to list more than one IPFS record? I can't think of one. (I think we should probably remove support for multiple Tor records as well, since Tor has its own, better, tooling for this, but that'll be its own PR.)